modules/ROOT/pages/afm-create-monitor.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can create a monitor by filling in fields if you do not know how to write tests in the Behavior Driven Development (BDD) test-writing language or how to use the BAT CLI. This method of creating a monitor creates a test suite that tests one or more endpoints, and schedules the test suite to run at intervals that you select.
+You can create a monitor by filling in fields if you do not know how to write tests in the Behavior Driven Development (BDD) test-writing language or how to use the BAT CLI. This method of creating a monitor creates a test that tests one or more endpoints, and schedules the test suite to run at intervals that you specify.
```

Comment:
"write tests in the Behavior Driven Development (BDD) test-writing language" - This is confusing, did we mean if a user wants to create simple ping tests or does not want to use BAT CLI to author complex test cases, user can use this UI to develop ping tests and advanced tests with assertions.

Reply:
@harshakirank , I'm referring to the language that is documented here in the AFM docs: https://docs.mulesoft.com/api-functional-monitoring/bat-bdd-reference